src/FSharp.Compiler.CodeDom/CodeGenerator.fs

```diff
@@ -1175,7 +1183,7 @@ let getMethodOverloads (membs:CodeTypeMemberCollection) =
 /// fields
 let generateField (c:CodeMemberField) =
   id
-  +> generateCustomAttrDecls c.CustomAttributes ""
+  +> generateCustomAttrDeclsForEnumField c.CustomAttributes ""
```

Comment:
Oops, this should not change. Confused ``generateField`` with ``generateEnumField``